robottelo/vm.py

```diff
@@ -498,6 +506,18 @@ def register_contenthost(self, org, activation_key=None, lce=None,
             )
             if auto_attach:
                 cmd += u' --auto-attach'
+        elif consumerid is not None:
+            if username is None and password is None:
+                username = settings.server.admin_username
+                password = settings.server.admin_password
+
+            cmd += u' --consumerid {0} --username {1} --password {2}'.format(
+                consumerid,
+                username,
+                password,
+            )
+            if auto_attach:
+                cmd += u' --auto-attach'
```

Comment:
you can strip the ```u``` from your string definitions, as we don't need to support python 2 anymore.

Reply:
from my point of view this should be done as one PR, where all `u'` definitions should be removed.

Reply:
or at least, I can clear the whole file in another PR, I don't think it's good idea to mix this stuff together